common/tasks/linters/yamllint/yaml-lint.yaml

```diff
@@ -15,6 +15,10 @@ spec:
     - name: git-revision
       description: The Git revision (commit SHA) from which the test pipeline is originating.
       type: string
+    - name: config-file
```

Comment:
remote-config-file

Reply:
why remote-config-file?
the step `clone-refs` will first clone  repo, in generally the config-file should be defined in product repo. they would like to use their own

Reply:
fixed